GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/AetherbladeHideout.cs

```diff
@@ -217,6 +218,14 @@ internal override List<PhaseData> GetPhases(ParsedEvtcLog log, bool requirePhase
 
         internal override void EIEvtcParse(ulong gw2Build, FightData fightData, AgentData agentData, List<CombatItem> combatData, IReadOnlyDictionary<uint, AbstractExtensionHandler> extensions)
         {
+            // Ferrous Bombs
+            var bombs = combatData.Where(x => x.DstAgent == 89640 && x.IsStateChange == ArcDPSEnums.StateChange.MaxHealthUpdate).Select(x => agentData.GetAgent(x.SrcAgent, x.Time)).ToList();
```

Comment:
add a Where(x => x.Type == AgentItem.AgentType.Gadget) before ToList() just to be on the safe side